content/en/real_user_monitoring/guide/rum-for-product-analytics.md

```diff
@@ -0,0 +1,376 @@
+---
+title: Use RUM & Session Replay for Product Analytics
+kind: guide
+description: Learn how to use RUM & Session Replay to monitor trends in user behavior and feature adoption.
+further_reading:
+- link: '/real_user_monitoring/explorer/'
+  tag: 'Documentation'
+  text: 'Learn about the RUM Explorer'
+- link: '/real_user_monitoring/'
+  tag: 'Documentation'
+  text: 'Learn how to visualize your RUM data'
+- link: '/real_user_monitoring/frustration_signals'
+  tag: 'Documentation'
+  text: 'Learn about frustration signals'
+- link: '/real_user_monitoring/session_replay'
+  tag: 'Documentation'
+  text: 'Learn about Session Replay'
+- link: '/dashboards/guide/powerpacks-best-practices/'
+  tag: 'Documentation'
+  text: 'Learn about powerpacks'
+---
+
+## Overview
+
+[RUM & Session Replay][1] allows you to monitor trends in consumer behavior and uncover answers to product usage questions about your web and mobile applications.
+
+This guide discusses several use cases to enrich your RUM & Session Replay data and provides example queries you can use to answer questions about product analytics.
+
+## Setup
+
+Once you have set up the Datadog RUM SDK, enrich your [browser][2] or mobile ([iOS][3] and [Android][4]) data with attributes to customize the data according to your use case. For example, adding contextual information allows you to [identify sessions that are tied to specific users][4].
+
+## Monitor page traffic and feature usage
+
+If you are interested in learning what buttons your users click on the most, you can track page traffic and the usage of buttons in your application. 
+
+1. Navigate to the [RUM Explorer][5] and select **Actions** from the dropdown menu next to the search query. 
+2. Enter `@view.name:/cart` in the search query and select the **Top List** visualization type.
+3. In the `by` field of the `Group into fields` section above, select **Action Name** from the group dropdown.
+
+This example displays the top actions on Shopist's `/cart` page.
+
+{{< img src="real_user_monitoring/guide/rum-for-product-analytics/actions_in_cart_page.png" alt="Search query for actions on Shopist's Cart page" style="width:90%;">}}
+
+Import this into a dashboard by copying the code snippet and selecting **Import dashboard JSON**: 
```

Comment:
For this and the other instructions like it... 

I clicked Copy on the code blocks, went to a new dashboard, and from the Settings menu, selected **Import dashboard JSON**, but got an error. 

What did work for me was, after creating the toplist in the Explorer, clicking Export > Export to dashboard and selecting a db to put it in. 

But note that this only works on the Demo org -- which I don't think customers have access to. Even setting up the query you describe in step 2 doesn't work outside Demo -- you'd have to tell them to choose a view name in their particular web app.

So I think the instructions in this guide have to be generalized a bit to clarify when you're telling them to do something exactly, vs when they'll see something slightly different in their actual org.